include/glibby/primitives/point.h

```diff
@@ -10,12 +10,21 @@ namespace glibby
 	struct Point
 	{
 		int axis = N;
-		T points[N];
+		T coord[N];
 
-		Point() { for (size_t i = 0; i < N; i++) { points[i] = 0; } };
-		explicit Point(T all) { for (size_t i = 0; i < N; i++) points[i] = all; }
-		explicit Point(T arr[N]) { for (size_t i = 0; i < N; i++) points[i] = arr[i]; }
-		explicit Point(std::vector<T> arr) { for (size_t i = 0; i < N; i++) points[i] = arr[i]; }
+		Point() { for (size_t i = 0; i < N; i++) { coord[i] = 0; } };
+		explicit Point(T all) { for (size_t i = 0; i < N; i++) coord[i] = all; }
+		explicit Point(T arr[N]) { for (size_t i = 0; i < N; i++) coord[i] = arr[i]; }
+		explicit Point(std::vector<T> arr) { for (size_t i = 0; i < N; i++) coord[i] = arr[i]; }
+
+		bool operator==(Point<T, N> p) {
```

Comment:
curly braces should go on a new line, and p should be a const reference. also, it is idiomatic to make this overload a non-member function. You don't have to make it a non-member function ~~, but you should move the definition into the cpp file~~ forgot it was templated :)